pkg/minikube/machine/drivers/none/none.go

```diff
@@ -46,8 +46,15 @@ func NewDriver(hostName, storePath string) *Driver {
 	}
 }
 
-// PreCreateCheck checks that VBoxManage exists and works
+// PreCreateCheck checks for correct priveledges and dependencies
 func (d *Driver) PreCreateCheck() error {
+	// check that docker is on path
+	_, err = exec.LookPath("docker")
```

Comment:
It is unclear the best way to check this.  A user can have the daemon running but not at the default port, then if you attempted this check with say, 'docker ps' it would return false even though the user might be passing the correct daemon configuration to the kubelet with extra-config